cypress/e2e/full/P2P/filterPaymentMethod.cy.js

```diff
@@ -34,11 +34,11 @@ function verifyAllPaymentMethod(orderTab, PM1, PM2) {
 }
 
 function addOrderWithPM() {
-  cy.c_addSellOrderDetails('Other', '10', '0.1', '1', '2')
+  cy.c_addSellOrderDetails('Other', '10', '0.1', '1', '2', 'fixed')
```

Comment:
Suggestion (Not a change request): can we update this command to take a single object as a parameter instead of passing multiple mandatory inputs 

the reason I am saying this is because looking at this command I dont know what is 10 what is 0.1 etc 

if we pass the parameters in a object  then it would be more understandable 
```suggestion
  cy.c_addSellOrderDetails({
  var1:'Others',
  var2:'10',
  var3:'1',
  var4:'2',
  var5:'fixed'
  })
```